circle.yml

```diff
@@ -26,6 +26,7 @@ dependencies:
   override:
     - ./scripts/time-cmd.sh ./gradlew --profile --parallel resolveConfigurations
     - scripts/circle-ci/pr-changelog-status-check.sh
+    - ./gradlew --stacktrace generateLock verifyDependencyLocksAreCurrent
```

Comment:
`--profile` and `--parallel` may be quite nice though (Probably with a `./scripts/time-cmd.sh` at the beginning)